api/v1beta1/helmchart_types.go

```diff
@@ -45,9 +45,15 @@ type HelmChartSpec struct {
 	// +required
 	Interval metav1.Duration `json:"interval"`
 
-	// Alternative values file to use as the default chart values, expected to be a
-	// relative path in the SourceRef. Ignored when omitted.
+	// Alternative list of values files to use as the default chart values,
```

Comment:
If I'm not mistaken, I think the order in which the files are specified is important as well, considering that map merging is used. I think this fact also needs to be specified in the documentation.

Reply:
Basically, what is suggested here? https://github.com/fluxcd/source-controller/pull/305#discussion_r591304540

Reply:
Yes, looks like I missed your comment, that's exactly what I mean.

Reply:
Any suggestions are welcome!